src/pages/components/content-block-simple.mdx

```diff
@@ -32,9 +33,22 @@ Content block &mdash; simple with aside elements is also used for introducing pa
 
 ![Image of content block simple with aside elements](../../images/pattern/content-block-simple/content-block-simple-with-aside-elements.jpg)
 
+## Content guidance
+
+| Field | Field type | Required   | Cardinality | Max character size (English / translated) |
+| -------- | --------- | ---------- | ---------- | ----------|
+| Headline | Short copy | Yes | 1 | XXS (40 / 55)|
+| TOC headline override | Short copy | No | 1 | Mini (25 / 35)|
+| Paragraph | Long copy | Yes | 1 | XXXXL (1600 / 2400)|
+| [Image with caption](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/image-with-caption) or [Video player](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/video-player) | Component | No | 1 | - |
+| [CTA (card)](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/cta) | Component | No | 1 | - |
+| [Link list](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/link-list) | Component | No | 1 | - |
```

Comment:
Hmm... I might need to update functional specs to align with this. I still don't believe this should be apart of this component personally. I think it breaks the system we've created up to this point.

Reply:
@photodow If we want to remove the link list integration, then I think we definitely need to get rid of the "Variation" section from this page.